roles/common/tasks/redhat_package_cache.yml

```diff
@@ -4,33 +4,41 @@
 #  lineinfile: dest=/etc/yum.conf line="proxy=http://{{ package_cache_server }}:3128" insertafter="^[main]"
 
 # XXX Better way to remove this plugin? XXX
-- name: remove fastestmirror plugin
-  file: path=/etc/yum/pluginconf.d/fastestmirror.conf state=absent
+#- name: remove fastestmirror plugin
```

Comment:
I did it beacuse i had problems but IDK if its the ight thing, same for the reset of the file